tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -0,0 +1,228 @@
+        """Check katello-certs-check generate correct command
+
+        :id: 4c9e4c6e-8d8e-4953-87a1-09cb55df3adf
+
+        :steps:
+
+            1. Generate custom certs
+            2. Run katello-certs-check with the required valid arguments
+               katello-certs-check -c CERT_FILE -k KEY_FILE -r REQ_FILE
+               -b CA_BUNDLE_FILE
+            3. Assert the output has correct commands with options
+
+        :expectedresults: Katello-certs should generate correct commands with
+                          options.
+
+        :caseautomation: notautomated
+        """
+
+    @destructive
+    @stubbed()
+    def test_positive_update_katello_certs(self):
+        """Update certificates on a currently running satellite installation
+
+        :id: 0ddf6954-dc83-435e-b156-b567b877c2a5
+
+        :steps:
+
+            1. Generate custom certs
+            2. Run katello-certs-check with the required valid arguments
+            3. Update satellite with custom certs
+            4. Assert output is having correct capsule-certs-generate commands
+
+        :expectedresults: Katello-certs should be updated.
+
+        :caseautomation: notautomated
+        """
+
+    @destructive
+    @stubbed()
+    def test_positive_update_capsule_Katello_certs_using_absolute_path(self):
+        """Update certificates on a currently running internal capsule
+
+        :id: 72024757-be6f-49f0-8b88-c57c83f5e7e9
+
+        :steps:
+
+            1. Generate custom certs
+            2. Run katello-certs-check with the required valid arguments
+            3. Run  capsule-certs-generate with custom certs and absolute path
+               for --certs-tar
+
+        :expectedresults: Katello-certs should be updated.
+
+        :caseautomation: notautomated
+        """
+
+    @destructive
+    @stubbed()
+    def test_positive_update_capsule_Katello_certs_using_relative_path(self):
+        """Update certificates on a currently running internal capsule
+
+        :id: 50df0b87-d2d3-42fb-86d5-988ebaaa9ba3
+
+        :steps:
+
+            1. Generate custom certs
+            2. Run katello-certs-check with the required valid arguments
+            3. Run  capsule-certs-generate with custom certs and relative path
+               for --certs-tar
+
+        :expectedresults: Katello-certs should be updated.
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier1
+    def test_negative_check_expiration_of_certificate(self):
+        """Check expiration of certificate
+
+        :id: 0acce44f-ebe5-42e1-a74b-3d4d20b97946
+
+        :steps:
+
+            1. Use expired certificate
+            2. Run katello-certs-check with the required arguments
+
+        :expectedresults: Checking expiration of certificate check should fail.
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier1
+    def test_negative_check_ca_bundle(self):
+        """Check ca bundle file has correct data
```

Comment:
if it is negative scenario then it is more `Check ca bundle file that contains invalid data`